recipes/rapidcheck/all/conanfile.py

```diff
@@ -33,32 +37,23 @@ class RapidcheckConan(ConanFile):
         "enable_gtest": False,
     }
 
-    generators = "cmake", "cmake_find_package"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
-    @property
-    def _is_msvc(self):
-        return str(self.settings.compiler) in ["Visual Studio", "msvc"]
+    def _min_cppstd(self):
+        return 11
 
     def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
     def requirements(self):
         if self.options.enable_catch:
```

Comment:
Could you try to bump the versions of these dependencies? `gtest` looks like a clean bump, but catch might be a bit trickier, let me know if you need any help with that!

Reply:
I bumped catch2 to latest 2.x version and gtest to latest c++11 compatible one.